www/js/survey/enketo-survey.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.module('emission.survey.enketo.launch', [
+  'emission.services',
+  'emission.enketo-survey.services',
+  'emission.plugin.logger',
+])
+.controller('EnketoSurveyCtrl', function($scope, $state, $stateParams, $rootScope,
+  $ionicPopup, EnketoSurvey
+) {
+  if (!$rootScope.previousState){
```

Comment:
not sure why you need this check

Reply:
That is only for development purpose. It appears that if I opened this page and the code got reloaded, the back button disappear. This help fix it by navigate back to diary page. It's not perfect but it works on the development environment and it should not interfere with the live deployment.